src/shared/lib/connectors/services/water/GaugingStationsService.js

```diff
@@ -0,0 +1,10 @@
+const ServiceClient = require('../ServiceClient');
+const config = require('internal/config');
+class GaugingStationsService extends ServiceClient {
+  getGaugingStationLicences (gaugingStationId) {
+    const id = gaugingStationId;
```

Comment:
Is there any need to define id here?

Reply:
In other words, is it not the same as `const url = `${config.services.water}/gauging-stations/${gaugingStationId}/licences`;`